pkg/kubelet/container/runtime.go

```diff
@@ -428,12 +428,6 @@ type RunContainerOptions struct {
 	ReadOnly bool
 	// hostname for pod containers
 	Hostname string
-	// EnableHostUserNamespace sets userns=host when users request host namespaces (pid, ipc, net),
-	// are using non-namespaced capabilities (mknod, sys_time, sys_module), the pod contains a privileged container,
-	// or using host path volumes.
-	// This should only be enabled when the container runtime is performing user remapping AND if the
-	// experimental behavior is desired.
-	EnableHostUserNamespace bool
```

Comment:
Why removing this from the struct? Is this still supported?

Reply:
It's not needed anymore after the changes in this PR. The logic to calculate the user namespace to use is moved to the kuberuntime completely.

Reply:
Makes sense, thanks!